pkg/cloud/volume_limits_table.go

```diff
@@ -501,7 +497,7 @@ var volumeLimits = map[string]volumeLimit{
 	"m8a.large":           {32, "dedicated"},
 	"m8a.medium":          {32, "dedicated"},
 	"m8a.metal-24xl":      {64, "dedicated"},
-	"m8a.metal-48xl":      {79, "dedicated"},
+	"m8a.metal-48xl":      {79, "shared"},
```

Comment:
Similarly, I've confirmed this actually is what the API returns, idk why we had a different value.